lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
+
+var chalk,
+    table,
+    pluralize,
+    drawReport,
+    drawTable;
+
+chalk = require("chalk");
+table = require("table").default;
+pluralize = require("pluralize");
```

Comment:
Do we really need a dependency for this as I have seen in other area where we have just handled this by our own?